backup_and_restore/application_backup_and_restore/features-and-plugins/oadp-configuring-velero-plugins.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/oadp-features-plugins.adoc
+[id="oadp-configuring-velero-plugins"]
+= About OADP Velero plugins
+include::_attributes/common-attributes.adoc[]
+:context: oadp-features-plugins
+:_content-type: CONCEPT
+
+toc::[]
+
+You can configure two types of plugins when you install Velero:
+
+* Default cloud provider plugins
+* Custom plugins
+
+Both types of plugin are optional, but most users configure at least one cloud provider plugin.
+
+== Default Velero cloud provider plugins
+
+You can install any of the following default Velero cloud provider plugins when you configure the `oadp_v1alpha1_dpa.yaml` file during deployment:
+
+* `aws` (Amazon Web Services)
+* `gcp` (Google Cloud Platform)
+* `azure` (Microsoft Azure)
+* `openshift` (OpenShift Velero plugin)
+* `csi` (Container Storage Interface)
+* `kubevirt` (KubeVirt)
```

Comment:
kubevirt should be removed, it's not a native velero plugin.  It's openshift only